alvr/session/src/settings.rs

```diff
@@ -27,6 +27,24 @@ pub enum NvencPreset {
     LowLatencyHighPerformance = 2,
 }
 
+#[repr(u32)]
+#[derive(SettingsSchema, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase", tag = "type", content = "content")]
+pub enum H264QualityPreset {
+    Quality = 2,
+    Balanced = 0,
+    Speed = 1,
+}
+
+#[repr(u32)]
+#[derive(SettingsSchema, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase", tag = "type", content = "content")]
+pub enum H265QualityPreset {
+    Quality = 0,
+    Balanced = 5,
+    Speed = 10,
+}
+
```

Comment:
Can these two enums be merged into one? Use 0, 1, 2 for the indices and then get the AMF related code on the C++ side. Maybe it's worth merging this also with the nvenc enum above?

Reply:
I'm not sure it would be right to merge it with NvencPreset, as the encoder quality preset settings are handled a bit differently by NVEnc and AMF. NvencPreset configures the whole encoder, and equivalent setting in AMF would be Usage (AVC_USAGE and HEVC_USAGE).